infrastructure/metrics/src/main/java/tech/pegasys/teku/infrastructure/metrics/Validator/ValidatorDutyMetricsSteps.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Consensys Software Inc., 2023
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.infrastructure.metrics.Validator;
+
+public enum ValidatorDutyMetricsSteps {
+  TOTAL("total"),
+  CREATE_TOTAL("create_total"),
+  CREATE("create"),
+  SIGN("sign"),
+  SEND("send");
```

Comment:
I wonder if it would be valuable to add a brief description of CREATE_TOTAL to help others understand its meaning?

Reply:
in code? it might help devs i guess but does nothing to help a user... if you look at where its used its when calling validatorApiChannel... seems fairly obvious?